sansa-ml-spark/src/test/scala/net/sansa_stack/ml/spark/clustering/algorithms/LocalitySensitiveHashingTest.scala

```diff
@@ -0,0 +1,26 @@
+package net.sansa_stack.ml.spark.clustering.algorithms
+
+import com.holdenkarau.spark.testing.DataFrameSuiteBase
+import org.apache.jena.riot.Lang
+import org.apache.spark.rdd.RDD
+import org.scalatest.FunSuite
+
+import net.sansa_stack.ml.spark.clustering.algorithms._
+
+class LocalitySensitiveHashingTest extends FunSuite with DataFrameSuiteBase {
+  
+  import net.sansa_stack.rdf.spark.io._
+
+  val path = getClass.getResource("/Cluster/testSample.nt").getPath
+  val graphframeCheckpointPath = "/sansa-ml-spark_2.11/src/main/scala/net/sansa_stack/ml/spark/clustering/algorithms/graphframeCheckpoints"
```

Comment:
You can also use the `getResource(..)` method for the checkpoints as well or you explicitly need to write it on the class level? which is a bit strange to me. Shall we consider moving it as a resource? when it is needed to be generated?